README.md

```diff
@@ -1,21 +1,46 @@
 # Destiny - Cross-platform Magic Wormhole graphical client
 
-⚠ This project is still a work in progress, use at your own risk ⚠
+## An end-to-end encrypted app to share files without revealing your identity 
 
-Easily send and receive files without signup or loss of privacy. Based on the
-[wormhole-william](https://github.com/LeastAuthority/wormhole-william)
+Destiny is a secure file transfer application that allows people to transfer files without needing to reveal their identities to each other or the service provider. All files are end-to-end encrypted, meaning no one except the sender and the receiver can decrypt the contents. Users select a file on their device and then share the generated code with the intended recipient for safe delivery. No sign-up is needed.
+
+Key security features: 
+
+- **Identity-less**: no need to disclose identity information (such as name, email address, or phone number) to be able to transfer files.
+- **End-to-end encryption**: files are end-to-end encrypted and only the sender and recipient can read them.
+- **Peer-to-peer** file sharing: Destiny attempts to make a direct network connection to the other party. If both parties are on the same local network they should connect without any traffic leaving that network, for example. When this isn’t possible (e.g. if neither party has a public IP address) then our relay server is used (however, that server sees only encrypted packets).
+- **Full-strength keys**: although our codes are short and human-memorable, they are part of an online “Password Authenticated Key Exchange” (PAKE) which only allows a single guess – and yields a 256-bit full-strength symmetric key.
+
+
+Based on the [wormhole-william](https://github.com/LeastAuthority/wormhole-william)
 implementation of the [Magic Wormhole protocol](https://github.com/magic-wormhole/magic-wormhole).
-This application relies on a plugin that wraps the wormhole-william client
-with a Dart API. The plugin is implemented [here](https://github.com/LeastAuthority/dart_wormhole_william).
+This application relies on [a plugin that wraps the wormhole-william client](https://github.com/LeastAuthority/dart_wormhole_william)
+with a Dart API.
 
 Supported platforms
 
-- [x] Linux
+- [x] Linux (amd64)
 - [x] Windows
-- [x] OSX
+- [x] macOS
 - [x] Android
 - [ ] iOS
 
+## Usage
+
+### Installation
+
+You can find detailed instructions on how to install applications on various platforms [here](https://github.com/LeastAuthority/destiny/blob/main/doc/installation.md)
```

Comment:
I always prefer to avoid "here" link-text.

Maybe "We have [detailed instructions on how to install] the application on various platforms." (with the stuff in "[" being the link).